sdk/operationalinsights/arm-operationalinsights/src/operations/dataExports.ts

```diff
@@ -144,28 +169,6 @@ export class DataExports {
       deleteMethodOperationSpec,
       callback);
   }
-
-  /**
-   * Create or update a data export.
-   * @param resourceGroupName The name of the resource group. The name is case insensitive.
-   * @param workspaceName The name of the workspace.
-   * @param dataExportName The data export rule name.
-   * @param parameters The parameters required to create or update a data export.
-   * @param [options] The optional parameters
-   * @returns Promise<msRestAzure.LROPoller>
-   */
-  beginCreateOrUpdate(resourceGroupName: string, workspaceName: string, dataExportName: string, parameters: Models.DataExport, options?: msRest.RequestOptionsBase): Promise<msRestAzure.LROPoller> {
```

Comment:
Looks like a long running operation has been converted to a normal operation. Are we sure that this was intentional?